src/elements/List/ListHeader.js

```diff
@@ -6,19 +6,19 @@ import {
   childrenUtils,
   createShorthandFactory,
   customPropTypes,
-  getElementType,
+  ElementType,
   getUnhandledProps,
   META,
+  withComputedType,
 } from '../../lib'
 
 /**
  * A list item can contain a header.
  */
-function ListHeader(props) {
+const InnerListHeader = (props) => {
   const { children, className, content } = props
   const classes = cx('header', className)
-  const rest = getUnhandledProps(ListHeader, props)
-  const ElementType = getElementType(ListHeader, props)
+  const rest = getUnhandledProps(InnerListHeader, props, { passAs: true })
```

Comment:
Another nasty thing. `as` should be passed to `ElementType`, but not all our components are updated.

I've already add `ignoreProps` option [`babel-plugin-transform-react-handled-props` ](https://github.com/layershifter/babel-plugin-transform-react-handled-props) plugin, but we can't enable it until we will update all components

Reply:
@layershifter do we have a list of the components that still need to get updated somewhere? I am assuming this exists in another conversation/PR somewhere.

Reply:
After this PR will be merged I will update #1321 with list for update. I don't want to do this in single PR because there will be too many changes.